certify-service/src/test/resources/schema.sql

```diff
@@ -45,3 +45,11 @@ CREATE TABLE  IF NOT EXISTS key_store(
 	del_dtimes timestamp,
 	CONSTRAINT pk_keystr_id PRIMARY KEY (id)
 );
+
+CREATE TABLE IF NOT EXISTS svg_render_template (
```

Comment:
can we rename table as svg_template?

Reply:
Changed the table name to svg_template and all related places like entity and repository